spec/requests/organizations/organization_members_controller_spec.rb

```diff
@@ -0,0 +1,183 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Organizations::OrganizationMembersController, type: :request do
+  context 'when a organization admin is signed in' do
+    let(:organization) { create :organization }
+    let(:organization_admin) { create :organization_member, organization: organization }
+
+    before { sign_in organization_admin.user }
+
+    describe '#index' do
+      subject { response }
+
+      let(:outside_organization) { create :organization }
+      let(:perform_request) { get organizations_members_path }
+      let!(:outside_organization_member) do
+        create(:organization_member,
+               :without_login,
+               organization: outside_organization,
+               first_name: 'Outside',
+               last_name: 'Organization Collaborator')
+      end
+
+      before do
+        create(:organization_member, :without_login, organization: organization)
+        perform_request
+      end
+
+      it 'returns a list of organization members inside the organization' do
+        expect(response).to be_successful
+        expect(response.body).to include(*organization.organization_members.pluck(:first_name))
+        expect(response.body).not_to include(
+          outside_organization_member.first_name,
+          outside_organization_member.last_name
+        )
```

Comment:
Why u spec only first_name from inside organization member but first and last from outside?